packages/react-components/react-avatar/src/stories/AvatarGroupDefault.stories.tsx

```diff
@@ -1,4 +1,36 @@
 import * as React from 'react';
 import { AvatarGroup, AvatarGroupProps } from '../index';
```

Comment:
_nit:_ You could split this by using the specific files to be able to use the `import type` pattern

```suggestion
import { AvatarGroup } from '../components/AvatarGroup/AvatarGroup';
import type { AvatarGroup } from '../components/AvatarGroup/AvatarGroup.types';
```

Or you could even keep using `index` but still split them:

```suggestion
import { AvatarGroup } from '../index';
import type { AvatarGroup } from '../index';
```